data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
Firefox 28 was removed by you from browser list for `non-standard` tab. Look at diff starting at line 79. There is only `firefox20` (which includes version from 20 to 29) and `firefox28` anymore in your branch